Reflections/Wk1.txt

```diff
@@ -0,0 +1,5 @@
+1. This week I have finished up research with my APIs and have chosen the Yelp API. After that I watched some Youtube Videos and discovered a block code website much like AppInventor. After that I was able to create a basic prototype to show off in class. I created two pages, the menu page with a distance slider and the results page which is abloe to display the name of the restaurant, the phone number,  a picture of the restaurant, a reroll button and a more info button.
+2. The biggest challenge I faced was learning the new website. Although it is very similar to AppInventor, it still has new  features that I wasn't sure how worked. I used a lot of the websites tutorials to figue out which each thing did.

```

Comment:
It is always hard to get used to a new software with new features! Tutorials are always a good idea for figuring things out. I have not used Thunkable myself but if you need another pair of eyes looking into anything specific let me know!